templates/password_reset.txt

```diff
@@ -0,0 +1,5 @@
+{{t "Password Reset Successfully" }}
+{{t "Your Firefox Account password has been successfully reset." }}
+
+{{t "This is an automated email; if you didn’t reset the password to your Firefox Account, you should reset it immediately at %(resetLink)s" }}
+{{t "For more information, please visit https://support.mozilla.org" }}
```

Comment:
I'm not sure the top-level SUMO page will be much help, should we consider a dedicted article for this?